kinto/plugins/default_bucket/__init__.py

```diff
@@ -101,14 +100,11 @@ def resource_create_object(request, resource_cls, uri):
         raise_invalid(resource.request, **error_details)
 
     data = {'id': obj_id}
-    try:
-        obj = resource.model.create_record(data)
-        # Since the current request is not a resource (but a straight Service),
-        # we simulate a request on a resource.
-        # This will be used in the resource event payload.
-        resource.postprocess(data, action=ACTIONS.CREATE)
-    except storage_exceptions.UnicityError as e:
-        obj = e.record
+    obj = resource.model.create_record(data, ignore_conflict=True)
```

Comment:
```python
# We use ignore_conflict here because each user can only ever have one default bucket, and it is created implicitly.
# If we got a conflict, it's from two requests by the same user, both creating a bucket.
# This isn't an error since creating the default bucket is idempotent.
```